autovalet_goal_generation/src/autovalet_goal_generation/goal_generator.py

```diff
@@ -44,30 +46,65 @@ def generate_goal_from_egoline(self, ego_line, egoline_frame):
         '''
 
         # ref: https://stackoverflow.com/questions/2298390/fitting-a-line-in-3d/2333251#2333251
+        ego_line         = ego_line[ego_line[:, 2].argsort()]
```

Comment:
why are we sorting here?

Reply:
forgot to remove. I was initially trying to take the one-third point along the ego line.

Reply:
removed.